### ISSUE TITLE
Mouse scroll wheel not working

### ISSUE THREAD

**Issue by antoinetran** [neutrinolabs/xrdp#324]
Hi,

I am using Red Hat 6.5 with rdesktop 1.8.2 (tested with 1.8.3 too) and xrdp 0.5.0-0.13.el6 (tested with 0.6.1-3.el6 too). I tried to connect to the server using rdesktop but the scroll wheel does not work. xev does not show any reactions when scrolling up or down. But using middle button works. What do I have to do for the server to receive wheel events?

Here are some useful details:
a) Windows native rdesktop client -> linux VM xrdp 0.5.0 scroll OK
a) Linux rdesktop 1.8.2 -> Windows VM scroll OK

According to the rdesktop client developper (source [here](https://github.com/rdesktop/rdesktop/issues/10), the issue is with the xrdp implementation, not supporting "slow-path".

I haven't tried with latest xrdp  0.6.1-4.el6, from epel repo.

Thank you.

**Comment by itamarjp** [comment]
Can you try with xrdp building the sources from git ?

**Comment by antoinetran** [comment]
I tried compiling it with --prefix, copying the compiled files in a VM. I had lots of trouble making it start. The service use new Lsb interface I believe, not used in RedHat 6. So I replaced theses functions with echo or start-daemon, but it didn't work. If I start manually (/etc/xrdp/xrdp.sh start) , I can see a port in LISTEN state. When I rdesktop to it, I have segmentation fault... So it's quite complicated...
